tests/src/cmds/core/test_user.py

```diff
@@ -1,9 +1,37 @@
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
 from src.cmds.core import user
+from tests import helpers
 
 
 class TestUserCog:
     """Test the `User` cog."""
 
+    @pytest.mark.asyncio
+    async def test_kick_success(self, ctx, bot):
+        ctx.user = helpers.MockMember(id=1, name="Test Moderator")
+        user_to_kick = helpers.MockMember(id=2, name="User to Kick", bot=False)
+        ctx.guild.kick = AsyncMock()
+        bot.get_member_or_user = AsyncMock(return_value=user_to_kick)
+
+        # Mock the DM channel
+        user_to_kick.send = AsyncMock()
+        user_to_kick.name = "User to Kick"
+
+        with patch('src.cmds.core.user.add_evidence_note', new_callable=AsyncMock) as add_evidence_mock, \
+                patch('src.cmds.core.user.member_is_staff', return_value=False):
```

Comment:
Very minor can we instead use parenthesis for improved readability?
```suggestion
        with (
            patch('src.cmds.core.user.add_evidence_note', new_callable=AsyncMock) as add_evidence_mock, 
            patch('src.cmds.core.user.member_is_staff', return_value=False
        ):
```

Reply:
Of course. My bad, sorry!